setup.py

```diff
@@ -361,7 +361,7 @@ def run(self):
       classifiers=trove_classifiers,
       # We support Python 2.7, and we're working on support for 3.6 (the
       # highest version that PyPy currently supports).
-      python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, <3.7",
+      python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*",
```

Comment:
During a recent meeting, I expressed concern about this change. After some consideration, I believe this change is as safe as the existing declaration (where the project installs on Python versions where it fails to run, i.e. 3.6).